src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java

```diff
@@ -204,6 +204,10 @@ public static String buildAppVersionTag(Run<?, ?> build) {
         return build.getParent().getDisplayName() + "-" + build.getNumber();
     }
 
+    public static String buildAppVersionTag(Run<?, ?> build, String applicationName) {
```

Comment:
Is `build` ever `null` ?

Reply:
Should not be null. Build is supplied by Jenkins to the "perform" method.